zone/entity.cpp

```diff
@@ -5937,3 +5937,26 @@ void EntityList::DamageArea(
 		}
 	}
 }
+
+std::vector<NPC*> EntityList::GetFilteredNPCList(std::vector<uint32> npc_ids, bool is_exclude)
```

Comment:
I'd rather have two methods versus just having a bool that changes behavior. This goes against SRP (single responsibility principle)

Reply:
Understood. I figured it’d cut down on duplication of code. But easy enough to separate.